### ISSUE TITLE
Working backwards

### ISSUE THREAD

**Issue by lynrayy** [romanvolodin/dynamic_parent#10]
Plugin increases parent's impact instead of compensate it

https://youtu.be/tjUFQ55Mv1A

**Comment by romanvolodin** [comment]
It's hard to say what's wrong without .blend file. Can you share it?
Looks like you parenting an object that is already parented.

чт, 26 янв. 2023 г. в 20:40, lynrayy ***@***.***>:

**Comment by lynrayy** [comment]
> Looks like you parenting an object that is already parented.

Yes, it was. I thought this plugin allows also temporary UN-parent object (recalculate object's location dure animation somehow). It's important for me because this file can't be edited (this file is animation constructor for minecraft emote mode and all parents already set)

(i think you can close this thread)
